debian/control

```diff
@@ -1,47 +1,74 @@
-Source: seafile
+Source: seafile-daemon
 Section: net
 Priority: extra
-Maintainer: Lingtao Pan <freeplant@gmail.com>
+Maintainer: m.eik michalke <meik.michalke@hhu.de>
 Build-Depends:
-	debhelper (>= 7),
-	autotools-dev,
-	libssl-dev,
-	libsqlite3-dev,
-	intltool,
-	libglib2.0-dev,
-	libevent-dev,
-	uuid-dev,
-	libgtk2.0-dev,
-    qtbase5-dev,
-    libqt5webkit5-dev,
-    qttools5-dev
-Standards-Version: 3.9.4
+    debhelper (>= 7),
+    autotools-dev,
+    libssl-dev,
+    libsqlite3-dev,
+    intltool,
+    libglib2.0-dev,
+    libevent-dev,
+    uuid-dev,
+    libtool,
+    libcurl4-openssl-dev,
+    valac,
+    libjansson-dev,
+    python,
+    libfuse-dev,
```

Comment:
Please remove the deps on `libfuse-dev`, and add `--disable-fuse` in the configure flags. 

It is a legacy problem: fuse is only used in seafile server, and should be automatically disabled in `configure.ac`. I'll fix that later, but for now let's use `--disable-fuse` flag.